### ISSUE TITLE
why tensorflow 1.14 does not have contrib module?

### ISSUE THREAD

**Issue by jinfagang** [tensorflow/tensorflow#32229]
```
    hparams = tf.contrib.training.HParams(
AttributeError: module 'tensorflow' has no attribute 'contrib'
(env) [16:29:19] fagangjin:tf2_tacotron2_chinese git:(master*) $ python
Python 3.5.2 (default, Nov 12 2018, 13:43:14) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
t>>> tf.__version__
'1.14.0'
>>> 

```

Does this normal?

**Comment by ravikyram** [comment]
@jinfagang 
TF 1.14 has contrib module.Please, find the [gist ](https://colab.sandbox.google.com/gist/ravikyram/f3cc2d7dca34311af82f3e9cb77b2eff/untitled155.ipynb) here.Thanks!

**Comment by mihaimaruseac** [comment]
It exists, see following test:

```console
(4) mihaimaruseac@ankh:/tmp/gh/4$ pip install -q tensorflow==1.14 && python -c "import tensorflow as tf; print(tf.contrib.training)"
WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

<module 'tensorflow.contrib.training' from '/tmp/gh/4/local/lib/python2.7/site-packages/tensorflow/contrib/training/__init__.pyc'>
```

Are you using a pip installed tensorflow or one built from source? Did you alter the code?
